File: cmd_inventory.py
from db_manager import get_inventory

COMMANDS = ["مخزوني", "مقتنياتي"]

def handle(bot, message):
    if message.text not in COMMANDS:
        return

    uid = message.from_user.id
    inv = get_inventory(uid)
    if not inv:
        bot.reply_to(message, "📦 مخزونك فارغ حالياً")
        return

    text = "📦 مخزونك الحالي:\n" + "\n".join(f"- {item}" for item in inv)
    bot.reply_to(message, text)

File: cmd_inventory.js
// ملف: cmd_inventory.js
const COMMANDS = ["مخزوني", "مقتنياتي"];
const db_manager = require("./db_manager"); // استدعاء مدير قاعدة البيانات

async function handle(bot, msg) {
    if (!msg.text) return;

    const text = msg.text.trim();
    if (!COMMANDS.includes(text)) return;

    const uid = msg.from.id;
    const inv = await db_manager.getInventory(uid);

    if (!inv || inv.length === 0) {
        bot.sendMessage(msg.chat.id, "📦 مخزونك فارغ حالياً");
        return;
    }

    const invText = inv.map(item => `- ${item}`).join("\n");
    const response = `📦 مخزونك الحالي:\n${invText}`;
    bot.sendMessage(msg.chat.id, response);
}

module.exports = { handle };
